generatemanmenu.pl

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env perl
+
+# Output to stdout
+my $OutManMenu = \*STDOUT;
+
+# Make option menu current with most recent version of curl.
+# Output options to file, remove first line, and make page for option menu.
+my $curlPath = @ARGV[0];                    # store path to curl in variable
+`$curlPath/src/curl -h all > options.txt`;  # output options to text file
+`tail -n +2 options.txt > tempOptions.txt`; # remove first line, outputting to tempOptions.txt
+`mv tempOptions.txt options.txt`;           # move temp to file options.txt        
```

Comment:
Rather than to rely on a curl build that is up to date, and to run commands in a shell here, it would be smoother to use the `../cvssource/docs/cmdline-opts` directory which contains the source files from which the curl man page is generated. That data is also easier to parse as it uses a documented format.

Reply:
I should be able to do this. My approach as of now will be based on:
```
if grep -q 'Short: .*' option.d; then
	grep -oE 'Short: ([[:alnum:]_-]+)' option.d | sed -E 's|Short: (.*)+|-\1|'
else
	grep -oE 'Long: ([[:alnum:]_-]+)' option.d | sed -E 's|Long: (.*)+|--\1|'
fi
```